common/src/main/java/tachyon/exception/IsNotKeyValueStoreException.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.exception;
+
+/**
+ * The exception thrown when a URI exists in Tachyon but is not a key-value store.
+ */
+// TODO(cc): Find a way to move this into keyvalue package.
+public class IsNotKeyValueStoreException extends TachyonException {
```

Comment:
I propose to have `InvalidPathException` for keyvalue store open. The reason is you may input a totally valid path, but it is not registered as a keyvalure store at all. In this case, `InvalidPathException` is more proper; of course, if the URI given is not even a valid path, `InvalidPathException` should be probably returned